### DISCUSSION TITLE
建议添加一个代理链接下载文件的选项

### DISCUSSION THREAD

**Discussion by RamSong** [spencerwooo/onedrive-vercel-index#601] [Ideas]
Onedrive国际版有时会抽风
使用https://github.com/RamSong/onedrive-vercel-index/blob/main/dai.js
通过cloudflare进行反代，为下载保驾护航
希望作者适配一下
[设计.png](https://i.imgtg.com/2022/04/15/k37FY.png)
可以像这样设计
点击之后链接自动切换为代理链接
Thanks

**Comment by spencerwooo** [comment]
如果翻看之前的 issue 你可能会发现，我们有支持过通过 Vercel 代理部分文件下载，但是 Vercel 有限制所以最终废弃了。

通过 Cloudflare 代理下载确实是可以考虑的但是需要单独配置一个 Cloudflare 代理 API 并做鉴权，实际上跟本项目没有太大关系了，所以我先暂时标注 `wontfix`，但不排除后续另开一个 Cloudflare 代理的项目并给这个项目添加可选配置。

**Reply by RamSong** [reply]
OK

**Reply by spencerwooo** [reply]
目前的话，README 文件渲染的时候是走 Vercel 代理并且有 cache 层的。
